### PR TITLE
Add CONFIG.child_friendly switch

### PR DESCRIPTION
**Pull request by cstamas**
And it controls things like:
* presence of "Watch on YT" link
* presence of likes/stats
* presence of BTC/liberapay (ask their parents for this!) :)

### CODE REVIEW

**Comment by SamantazFox** [comment]
Can you please rebase you changes on current master?

**Comment by cstamas** [comment]
@SamantazFox Tried rebasing, but all my changes (except dor docker-compose) conflicted, and there are a LOT changes since, but I currently have no time to keep up on this.
Please reuse my changes freely (they are few lines really), if interested to merge it.

File: src/invidious/views/template.ecr [hunk 1]
```diff
@@ -115,18 +115,22 @@
...
-                    <div class="pure-u-1 pure-u-md-1-3">
-                        <i class="icon ion-logo-bitcoin"></i>
-                        BCH: <a href="bitcoincash:qq4ptclkzej5eza6a50et5ggc58hxsq5aylqut2npk">qq4ptclkzej5eza6a50et5ggc58hxsq5aylqut2npk</a>
-                    </div>
-                    <div class="pure-u-1 pure-u-md-1-3">
-                        <i class="icon ion-logo-usd"></i>
-                        <a href="https://liberapay.com/omarroth">Liberapay</a>
-                    </div>
+
+                    <% if !CONFIG.child_friendly %>
```
Comment (unixfox) [inline src/invidious/views/template.ecr:119]: how is bitcoin not considered child friendly?

File: src/invidious/views/watch.ecr [hunk 2]
```diff
@@ -126,7 +128,7 @@
                 <script src="/js/playlist_widget.js?v=<%= Time.utc.to_unix_ms %>"></script>
             <% end %>
 
-            <% if CONFIG.dmca_content.includes?(video.id) || CONFIG.disabled?("downloads") %>
+            <% if CONFIG.child_friendly || CONFIG.dmca_content.includes?(video.id) || CONFIG.disabled?("downloads") %>
```
Comment (unixfox) [inline src/invidious/views/watch.ecr:131]: why is a child not allowed to download any video? what if he/she wants to download an educational video in order to share it with his/her friends?

**Comment by cstamas** [comment]
Am closing this out as it is just generating spam about failed builds for me. Sorry, but cannot pick this up now, will reopen when I will be able to update.

**Pull request closed without merging**